ruby_event_store_rom_sql/README.md

```diff
@@ -1,7 +1,54 @@
-# RubyEventStore::ROM
+# RubyEventStore ROM Event Repository
 
- A ROM based implementation of events repository for [Ruby Event Store](https://github.com/RailsEventStore/rails_event_store).
+A ROM-based implementation of events repository for [Ruby Event Store](https://github.com/RailsEventStore/rails_event_store).
 
 This is an alternative ActiveRecord `EventRepository` used in `rails_event_store` gem.
 
-The Ruby Event Store adapter for [rom-sql](https://github.com/rom-rb/rom-sql) is `RubyEventStore::ROM::EventRepository`.
+This version of the ROM adapter uses [rom-sql](https://github.com/rom-rb/rom-sql) and will be updated to facilitate multiple backends for implementing other (non-SQL) data stores behind ROM's API interface.
+
+## Get started
```

Comment:
How do the user of ROM generate DB migration? if the assumption is that they use RES-ActiveRecord one then maybe it would be worth mentioning it?